modules/renter/contractor/contracts.go

```diff
@@ -74,7 +73,10 @@ func (c *Contractor) managedMarkContractsUtility() error {
 	c.mu.RLock()
 	hostCount := int(c.allowance.Hosts)
 	c.mu.RUnlock()
-	hosts := c.hdb.RandomHosts(hostCount+minScoreHostBuffer, nil)
+	hosts, err := c.hdb.RandomHosts(hostCount+minScoreHostBuffer, nil)
```

Comment:
that's a terrible name for that constant.

let's change it to `minHostBufferForScore` lol

Reply:
no.

how about `randomHostsBufferForScore`